src/ray/stats/metric_exporter.h

```diff
@@ -40,10 +40,22 @@ class MetricExporter final : public opencensus::stats::StatsExporter::Handler {
 
   static void Register(std::shared_ptr<MetricExporterClient> metric_exporter_client,
                        size_t report_batch_size) {
+    is_exporting_ = false;
```

Comment:
This could cause a problem if `Register` is called multiple times. It is under the assumption that stats::Init will be called only once per process (unless cleanup is called), and metricsExport::Register is only used inside Stats::Init